idaes/commands/tests/test_commands.py

```diff
@@ -149,7 +149,8 @@ def test_examples_cli_copy(runner, tempdir):
     assert result.exit_code == -1
     # local dir exists, no REPO_DIR in it
     src_dir = tempdir / "examples-dev"
-    src_dir.mkdir()
+    if not src_dir.exists():
```

Comment:
Random minor suggestion: here `src_dir.mkdir(exist_ok=True)` (works like `mkdir -p`) could be used instead of the `if` check.

Reply:
ok